modules/installation-uninstall-infra.adoc

```diff
@@ -0,0 +1,354 @@
+** Python 3
+** The following Python modules:
+*** ansible version 2.9.2 or compatible
+*** openstacksdk version 0.39.0 or compatible
+*** openstackclient version 4.0.0 or compatible
+** On a Red Hat Linux Enterprise 8 machine, you must enable a few repositories to access these modules.
+... On a command line, add these repositories:
++
+----
+$ sudo subscription-manager register # If not done already
+$ sudo subscription-manager attach --pool=$YOUR_POOLID # If not done already
+$ sudo subscription-manager repos --disable=* # If not done already
+
+$ sudo subscription-manager repos \
+  --enable=rhel-8-for-x86_64-baseos-rpms \
+  --enable=openstack-16-tools-for-rhel-8-x86_64-rpms \
+  --enable=ansible-2.8-for-rhel-8-x86_64-rpms \
+  --enable=rhel-8-for-x86_64-appstream-rpms
+----
+... Install the modules:
++
+----
+$ sudo yum install python3-openstackclient ansible python3-openstacksdk
+----
+
+... Ensure that the `python` command points to `python3`:
++
+----
+sudo alternatives --set python /usr/bin/python3
+----
+
+[TIP]
+====
+On a Fedora machine, run this command to install the required packages:
+----
+$ sudo dnf install python-openstackclient ansible python-openstacksdk
+----
+====
+
+.Procedure
+
+[NOTE]
+You may have the `common.yaml` and `inventory.yaml` playbooks left over from when you installed {product-title}. If you do, you can skip the first two steps of the procedure.
+
+. Insert the following content into a local file called `common.yaml`:
++
+[source,yaml]
+----
+- hosts: localhost
+  gather_facts: no
+
+  vars_files:
+  - metadata.json
+
+  tasks:
+  - name: 'Compute resource names'
+    set_fact:
+      cluster_id_tag: "openshiftClusterID={{ infraID }}"
+      os_network: "{{ infraID }}-network"
+      os_subnet: "{{ infraID }}-nodes"
+      os_router: "{{ infraID }}-external-router"
+      # Port names
+      os_port_api: "{{ infraID }}-api-port"
+      os_port_ingress: "{{ infraID }}-ingress-port"
+      os_port_bootstrap: "{{ infraID }}-bootstrap-port"
+      os_port_master: "{{ infraID }}-master-port"
+      os_port_worker: "{{ infraID }}-worker-port"
+      # Security groups names
+      os_sg_master: "{{ infraID }}-master"
+      os_sg_worker: "{{ infraID }}-worker"
+      # Server names
+      os_bootstrap_server_name: "{{ infraID }}-bootstrap"
+      os_cp_server_name: "{{ infraID }}-master"
+      os_compute_server_name: "{{ infraID }}-worker"
+      # Trunk names
+      os_cp_trunk_name: "{{ infraID }}-master-trunk"
+      os_compute_trunk_name: "{{ infraID }}-worker-trunk"
+      # Subnet pool name
+      subnet_pool: "{{ infraID }}-kuryr-pod-subnetpool"
+      # Service network name
+      os_svc_network: "{{ infraID }}-kuryr-service-network"
+      # Service subnet name
+      os_svc_subnet: "{{ infraID }}-kuryr-service-subnet"
+      # Ignition files
+      os_bootstrap_ignition: "{{ infraID }}-bootstrap-ignition.json"
+----
+
+. Insert the following content into a local file called `inventory.yaml`:
```

Comment:
We should probably say that this is the file where we expect the user to insert their customised values